c7n/filters/iamaccess.py

```diff
@@ -206,6 +214,12 @@ def handle_aws_sourcearn(self, s, op, key, values):
     def handle_aws_sourceip(self, s, op, key, values):
         return False
 
+    def handle_aws_sourcevpce(self, s, op, key, values):
+        return bool(set(map(_account, values)).difference(self.allowed_vpce))
```

Comment:
if the option isn't configured by the user, this will still flag it, from a compatibility perspective it should default to not flagging it if vpce endpoint is not defined.

Reply:
👍

Reply:
so, qq, what you're saying is... if `aws:sourcevpc(e)` is not in the `whitelist_conditions` then skip the check? or if their is no values passed in for whitelist_vpc(e)?

Reply:
if someone wants to maintain original behavior they can specify `aws:sourceVpc` and/or `aws:sourceVpce` in whitelist_conditions, if they exist, this function will never be called in the first place.  We were also originally incorrectly defaulting them as well.  they should have had a capital `V`.  

Another way to maintain backwards compatibility is to leave sourceVpc and sourceVpce as default values and update policies to remove them.

Reply:
Sorry, disregard this latest commit... I misread the handle_statment function.